README.md

```diff
@@ -257,6 +257,24 @@ Arguments:
     php ./cli.php manage:set-organization-maintenance-mode [--force/-f] <manage-token> <organization Id> <on/off> <hostname-suffix> <reason> <estimatedEndTime> 
     ```
 
+## Delete Sandboxes/Workspaces that were created by no longer active token id
```

Comment:
This can be set to `setHelp` so it's part of the command. Not nessecery to have it separately in README IMHO

```
    protected function configure()
    {
        $this
            ...
            ->setHelp(...)
            ...
```

Reply:
ah yeah, I was just following how all the others are done